### ISSUE TITLE
A11y issues : Thumbnail alt and Previous/Next buttons role

### ISSUE THREAD

**Issue by trystan2k** [MurhafSousli/ngx-gallery#468]
Hi !

#### What is the expected behavior?
We are using `axe-core` (https://github.com/dequelabs/axe-core) to test the accessibility of our website (that uses ngx-gallery) and we found 2 issues related to missing attributes.
We expect to not have any a11y errors when using ngx-gallery

#### What is the current behavior?
The current errors are:  in the thumbnails images, that miss the `alt` attribute (which is already set in the main image) and the previous and next button give us an error saying that the `i` element cannot have `aria-label` attribute without a `role` attribute.

<img width="868" alt="image" src="https://user-images.githubusercontent.com/447294/192753129-7d4f2aa3-3687-4396-81c5-0d1425ce4a2f.png">

#### What are the steps to reproduce?

Just create a new Angular project with ngx-gallery (the StackBlitz starter, for example) and add a unit test with axe-core
```axe
  .run()
  .then(results => {
    if (results.violations.length) {
      throw new Error('Accessibility issues found');
    }
  })
  .catch(err => {
    console.error('Something bad happened:', err.message);
  });```

#### What is the use-case or motivation for changing an existing behavior?
Making ngx-gallery more accessible to everyone.


#### Which versions are you using for the following packages?

Angular: 14.2.2
Angular CDK: 14.2.2
Angular CLI: 14.2.2
Typescript: 4.8.3
Gallery: 7.0.4


#### Is there anything else we should know?
I can provide a PR later with these fixes, if it is ok for you.
